.github/workflows/tests.yml

```diff
@@ -16,7 +16,11 @@ jobs:
       fail-fast: false
       matrix:
         php-version:
-          - "7.1"
+          - "7.4"
+          - "8.0"
+          - "8.1"
+          - "8.2"
+          - "8.3"
```

Comment:
No need to run the tests for all PHP versions twice (once with and once without coverage).

Reply:
Is there a problem running on PHP 8.4 ?

Reply:
Overall, there shouldn’t be a problem. I added it, and we’ll see in Actions